lib/python/rose/suite_engine_procs/cylc.py

```diff
@@ -76,6 +79,27 @@ def check_global_conf_compat(self):
                 raise SuiteEngineGlobalConfCompatError(
                     self.SCHEME, key, lines[0])
 
+    def check_suite_not_running(self, suite_name):
+        """Check if a suite is still running.
+
+        Args:
+            suite_name (str): the name of the suite as known by Cylc.
+        Raise:
+            SuiteStillRunningError: if suite is still running.
+        """
+        fname = self.get_suite_dir(suite_name, ".service", "contact")
+        try:
+            lines_str = open(fname).read()
+        except IOError:
+            return  # No contact file, suite not running
+        else:
+            extras = ["Contact info from: \"%s\"\n" % fname]
+            for line in lines_str.splitlines(True):  # splitlines keep ends
+                if line.split("=")[0] in self.CONTACT_KEYS:
+                    extras.append("\t%s" % line)
```

Comment:
This tab character only results in a single space on the terminal for me, perhaps use spaces instead?

```
$ rose suite-run
[INFO] chdir: /suite/path/
[FAIL] Suite "suite-name" appears to be running:
[FAIL] Contact info from: "/suite/run/dir/.service/contact"
[FAIL] 	CYLC_SUITE_HOST=suite-host
[FAIL] 	CYLC_SUITE_OWNER=user
[FAIL] 	CYLC_SUITE_PORT=12345
[FAIL] 	CYLC_SUITE_PROCESS=19836 python cylc/path suite-name
[FAIL] Try "cylc stop 'suite-name'" first?
```

Reply:
Replaced tab with spaces.